### PR TITLE
HomeWork

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: Lesson10/src/main/java/JDBC/OrderDaoImpl.java [hunk 1]
```diff
@@ -5,13 +5,55 @@
...
+        Connection connection = ConnectToDB.getConnection();
+        Set<Order> order  = new HashSet<>();
+        PreparedStatement preparedStatement = connection.prepareStatement("SELECT * FROM ORDERS");
+        ResultSet result = preparedStatement.executeQuery();
+
+        while (result.next()){
+            Order order1 = new Order(result.getBigDecimal("order_Num"), null, result.getDate("order_Date"), result.getString("mfr"),
+                    result.getBigDecimal("qty"), result.getBigDecimal("amount"));
+
+            PreparedStatement preparedStatement1 = connection.prepareStatement("SELECT * FROM PRODUCTS WHERE PRODUCT_ID =? ");
```
Comment (Cherkashyn) [inline Lesson10/src/main/java/JDBC/OrderDaoImpl.java]: Name of PreparedStatement variable could be better :)

File: Lesson10/src/main/java/JDBC/OrderDaoImpl.java [hunk 2]
```diff
@@ -5,13 +5,55 @@
...
+            preparedStatement1.setString(1,result.getString("product"));
+            ResultSet result1 = preparedStatement1.executeQuery();
+            while (result1.next()){
+                Product product = new Product( result1.getString("product_Id"), result1.getString("mfr_Id"),
+                        result1.getString("description"), result1.getBigDecimal("price"), result1.getBigDecimal("qty_On_Hand"));
+                order1.setProduct(product);
+            }
+            result1.close();
```
Comment (Cherkashyn) [inline Lesson10/src/main/java/JDBC/OrderDaoImpl.java]: you can also close preparedStatement1

File: Lesson10/src/main/java/JDBC/OrderDaoImpl.java [hunk 3]
```diff
@@ -5,13 +5,55 @@
...
+            order.add(order1);
+        }
+        result.close();
+        connection.close();
```
Comment (Cherkashyn) [inline Lesson10/src/main/java/JDBC/OrderDaoImpl.java]: connection is closed in last turn

File: Lesson10/src/test/java/JDBC/OrderDaoImplIntegrationTest.java [hunk 4]
```diff
@@ -39,4 +39,18 @@ public void findOrderByIdSecondTest()throws SQLException {
         assertEquals(order,ORDER);
 
     }
+
+    @Test
+    public void getAllOrders()throws SQLException {
```
Comment (SergeyPasko) [inline Lesson10/src/test/java/JDBC/OrderDaoImplIntegrationTest.java]: Also good test in our situation - cheak that both methods return same results

**Pull request merged**